python/nav/models/manage.py

```diff
@@ -2365,6 +2372,13 @@ class Sensor(models.Model):
     alert_type = models.IntegerField(
         db_column='alert_type', choices=ALERT_TYPE_CHOICES, null=True
     )
+    threshold_type = models.IntegerField(
+        db_column='threshold_type', choices=THRESHOLD_TYPE_CHOICES, null=True
+    )
+    threshold_alert_type = models.IntegerField(
+        db_column='threshold_alert_type', choices=ALERT_TYPE_CHOICES, null=True
+    )
+    threshold_for_oid = VarcharField(db_column='threshold_for_oid', null=True)
```

Comment:
This would make sense to be a ForeignKey to the Sensor object this is a threshold for, but it didnt seem very easy to do. The values for the sensor object that is to be created are set after discovery, but before any objects are actually made, so refering to a sensor object that doesnt exist yet seems difficult. But pairing this `threshold_for_oid` field with a property function `threshold_for` that gives the actual sensor seemed like the next best thing.